src/cmd/uds.go

```diff
@@ -27,11 +27,20 @@ var createCmd = &cobra.Command{
 	Args:    cobra.MaximumNArgs(1),
 	Short:   lang.CmdBundleCreateShort,
 	PreRun: func(cmd *cobra.Command, args []string) {
-		if len(args) > 0 && !zarfUtils.IsDir(args[0]) {
-			message.Fatalf(nil, "(%q) is not a valid path to a directory", args[0])
+		pathToBundleFile := ""
+		if len(args) > 0 {
+			if !zarfUtils.IsDir(args[0]) {
+				message.Fatalf(nil, "(%q) is not a valid path to a directory", args[0])
+			}
+			pathToBundleFile = args[0] + "/"
 		}
-		if _, err := os.Stat(config.BundleYAML); len(args) == 0 && err != nil {
-			message.Fatalf(err, "%s not found in directory", config.BundleYAML)
+		// Handle .yaml or .yml
+		if _, yamlErr := os.Stat(pathToBundleFile + config.BundleYAML); yamlErr == nil {
```

Comment:
I see the intent here, but `yamlErr` seems misleading as var name; it's not actually a YAML error

Reply:
Updated